src/database/build.sql

```diff
@@ -0,0 +1,36 @@
+BEGIN;
+DROP TABLE IF EXISTS users, consultant,posts CASCADE;
+
+
+CREATE TABLE users(
+    id SERIAL  PRIMARY KEY  ,
+    user_name VARCHAR(60) NOT NULL,
+    password TEXT NOT NULL
+
+);
+CREATE TABLE consultant(
+    id SERIAL  PRIMARY KEY  ,
+    full_name VARCHAR(60) NOT NULL,
+    user_name VARCHAR(60) NOT NULL,
+    password TEXT NOT NULL
+);
```

Comment:
@alaabadra @denaHS @AymanAlqoqa @naremanhilles @hshahwan 
I wanted to quickly mention an alternative design for user tables and web apps that have different "types" or "permissions" of users:
Instead of having two tables for two different types of users (like here, with `user` and `consultant` you could have one table for all `user`, and then have a list of different permissions that a user could have (or types that a user could be).  For this app, you could just have a `user_type` integer field, 0 means normal user, 1 means consultant, for example.  Or a `permissions` varchar field that could hold more types of information.
The benefits of this kind of design are:
1) you only have one user table that you have to query, for login
2) you can add more user types very easily, like for example you decide you want "admins" who can delete posts.
3) a person can change user types without having to create a new account, for example if a normal user wanted to become a consultant.

BUT all of that said, these tables that you created are going to work fine for your app - I just wanted to mention another design that gets used often for larger-scale user permission systems.